<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="58.542" tests="4" failures="3">
  <testsuite name="Root Suite" timestamp="2025-05-11T19:15:50" tests="0" file="cypress/e2e/features/schedule_enrollment.feature" time="0.000" failures="0">
  </testsuite>
  <testsuite name="Schedule Enrollment" timestamp="2025-05-11T19:15:50" tests="4" time="58.503" failures="3">
    <testcase name="Schedule Enrollment Successfully enroll in a schedule" time="0.000" classname="Successfully enroll in a schedule">
      <failure message="expected false to be true" type="AssertionError"><![CDATA[AssertionError: expected false to be true
    at Context.eval (http://localhost:3001/__cypress/tests?p=cypress/e2e/features/schedule_enrollment.feature:10782:89)

      + expected - actual:  failed to generate Mocha diff
]]></failure>
    </testcase>
    <testcase name="Schedule Enrollment View enrolled schedules" time="6.532" classname="View enrolled schedules">
    </testcase>
    <testcase name="Schedule Enrollment Attempt to enroll in a full schedule" time="0.000" classname="Attempt to enroll in a full schedule">
      <failure message="expected 28 to equal 30" type="AssertionError"><![CDATA[AssertionError: expected 28 to equal 30
    at Context.eval (http://localhost:3001/__cypress/tests?p=cypress/e2e/features/schedule_enrollment.feature:10870:37)

      + expected - actual

      -28
      +30
      ]]></failure>
    </testcase>
    <testcase name="Schedule Enrollment Attempt to enroll in a schedule that conflicts with another enrolled schedule" time="0.000" classname="Attempt to enroll in a schedule that conflicts with another enrolled schedule">
      <failure message="expected 2 to be below 2" type="AssertionError"><![CDATA[AssertionError: expected 2 to be below 2
    at Context.eval (http://localhost:3001/__cypress/tests?p=cypress/e2e/features/schedule_enrollment.feature:10955:39)

      + expected - actual

]]></failure>
    </testcase>
  </testsuite>
</testsuites>